### ISSUE TITLE
Some docs not found in search

### ISSUE THREAD

**Issue by agaulton** [chembl/user_interface#66]
If I search for CHEMBL3637841 in 'Documents' I don't get any results back. I suspect this could be because this document has no assays associated with it, but I can't be sure...

**Comment by mnowotka** [comment]
Yes, I can confirm there is a problem. @nclopezo - please take a look at this test: https://app.ghostinspector.com/tests/5710b06427176e4f01b642bd (you should get an email notification anyway). It fails on this step:

![screen shot 2016-10-11 at 09 59 39](https://cloud.githubusercontent.com/assets/837119/19264635/79c0acc6-8f99-11e6-9c41-cf30d20d72e5.png)

You can log in to ghost inspector and click "Watch video" to see the test executing step by step.

**Comment by nclopezo** [comment]
In DOCUMENT_SOURCES, CHEMBL3637841's SRC_ID is 34 (MMV Pathogen Box). 34 was not in the list of datasources in the web interface configuration. I added it here:

https://github.com/chembl/chembl_interface/pull/14
You can now search for CHEMBL3637841
https://wwwdev.ebi.ac.uk/chembl/

I also noticed that the search PL/SQL code was still using the table journal_articles. I removed its use from the code. I would like to check it again, and also the PL/SQL code for other searches, when the chembl23 schema is ready.

**Comment by agaulton** [comment]
Great thanks, please could we make sure we have added all new sources to this list?
